core/src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -1,41 +1,36 @@
-import com.google.api.services.cloudbilling.Cloudbilling
-import com.google.api.services.cloudbilling.model.ProjectBillingInfo
-import com.google.api.services.cloudresourcemanager.CloudResourceManager
-import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.rawls.model.CreationStatuses.Ready
-import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevels._
-import org.broadinstitute.dsde.rawls.monitor.CreatingBillingProjectMonitor.CheckDone
-import org.broadinstitute.dsde.rawls.monitor.{CreatingBillingProjectMonitor, BucketDeletionMonitor}
-import org.broadinstitute.dsde.rawls.monitor.BucketDeletionMonitor.DeleteBucket
-import org.broadinstitute.dsde.rawls.util.Retry
-
-import scala.concurrent.{Await, ExecutionContext, Future}
-import scala.concurrent.duration.Duration
-import com.google.api.client.googleapis.auth.oauth2.{GoogleCredential, GoogleClientSecrets}
-import com.google.api.client.googleapis.json.GoogleJsonResponseException
 import com.google.api.client.auth.oauth2.TokenResponseException
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets
+import com.google.api.client.googleapis.json.GoogleJsonResponseException
+import com.google.api.client.http.{HttpResponseException, InputStreamContent}
 import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.admin.directory.model.Group
 import com.google.api.services.storage.model.{Bucket, StorageObject}
+import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.rawls.dataaccess._
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{RawlsBillingProjectOperationRecord, TestDriverComponent}
+import org.broadinstitute.dsde.rawls.metrics.StatsDTestUtils
+import org.broadinstitute.dsde.rawls.model.CreationStatuses.Ready
+import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevels._
 import org.broadinstitute.dsde.rawls.model._
+import org.broadinstitute.dsde.rawls.monitor.BucketDeletionMonitor
+import org.broadinstitute.dsde.rawls.monitor.BucketDeletionMonitor.DeleteBucket
+import org.broadinstitute.dsde.rawls.util.{MockitoTestUtils, Retry}
+import org.scalatest.concurrent.Eventually
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 import spray.http.{OAuth2BearerToken, StatusCodes}
-import org.broadinstitute.dsde.rawls.dataaccess.slick.{RawlsBillingProjectOperationRecord, TestDriverComponent}
 
 import scala.collection.JavaConversions._
-import scala.concurrent.duration._
+import scala.concurrent.duration.{Duration, _}
+import scala.concurrent.{Await, Future}
 import scala.util.Try
 
-class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationTestConfig with Retry with TestDriverComponent with BeforeAndAfterAll with LazyLogging {
+class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationTestConfig with Retry with TestDriverComponent with BeforeAndAfterAll with LazyLogging with Eventually with MockitoTestUtils with StatsDTestUtils {
```

Comment:
I haven't actually run these integration tests -- I should probably figure out how.